apis/v1alpha1/gateway_types.go

```diff
@@ -527,8 +527,10 @@ type GatewayAddress struct {
 // AddressType defines how a network address is represented as a text string.
 // Valid AddressType values are:
 //
-// * "IPAddress"
-// * "NamedAddress"
+// * "IPAddress": an IPv4 or IPv6 address
+// * "NamedAddress": an opaque identifier, controller-dependent
```

Comment:
Nit: I think it's worth updating the text here to more closely reflect the descriptions below.

Reply:
done